core/types/types.cc

```diff
@@ -837,15 +841,11 @@ core::SymbolRef Types::getRepresentedClass(core::Context ctx, const core::Type *
     if (!ty->derivesFrom(ctx, core::Symbols::Module())) {
         return core::Symbols::noSymbol();
     }
+
     core::SymbolRef singleton;
-    auto *s = core::cast_type<core::ClassType>(ty);
```

Comment:
is this a code refactor that doesn't change behavior?
Can we land it separately?
(I'm having high standard for commit history of major typesystem changes, as I can spend hours doing archeology on it)

Reply:
Yes, I'll move the two refactoring commits out of this PR.